iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -537,7 +553,7 @@ static LogicalResult setSPIRVOpConfig(const spirv::TargetEnv &targetEnv,
         // Try to tile and vectorize first.
         std::array<int64_t, 2> workgroupXY = {32, 2};
         std::array<int64_t, 3> threadMNK = {8, 8, 4};
-        auto result = detail::setMatmulOpConfig(op, workgroupXY, threadMNK);
+        auto result = detail::setMatmulOpConfig(op, 32, workgroupXY, threadMNK);
```

Comment:
nit: maybe move 32 in a variable with a descriptive name or add a comment?

Reply:
done